build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 plugins {
-    kotlin("multiplatform") version "1.8.10" apply false
+    kotlin("multiplatform") version "1.8.21" apply false
```

Comment:
Probably not related to the change in this PR, do you think we can move this version into toml file instead?

Reply:
I can do that with the next PR. Thanks to Gradle 8.0 for a better version catalog.